### PR TITLE
Crefi changes for py3 compatibility

### PR DESCRIPTION
**Pull request by yogesh-mane**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

adding delay because crefi hang
deleted one module import for py3 compatibility for rhel 7 - changes are made in my fork

### CODE REVIEW

**Comment by yogesh-mane** [comment]
4.2-rhel-7 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1604982047588
4.2-rhel--8 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1604985902570

File: tests/cephfs/CEPH-11228.py [hunk 1]
```diff
@@ -197,6 +198,7 @@ def run(ceph_cluster, **kw):
                     dir_name,
                     0)
 
+            time.sleep(900)
```
Comment (rakeshgm) [inline tests/cephfs/CEPH-11228.py:201]: this accounts  to 15 minutes. Do you need such a long sleep. any reason why ?

File: tests/cephfs/CEPH-11228.py [hunk 2]
```diff
@@ -210,6 +212,7 @@ def run(ceph_cluster, **kw):
                     client_info['mds_nodes'])
                 for op in p:
                     return_counts, rc = op
+            time.sleep(900)
```
Comment (rakeshgm) [inline tests/cephfs/CEPH-11228.py:215]: same as above

File: tests/cephfs/cephfs_utils.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def _setup_crefi(node):
         # clone crefi repository
         node.exec_command(
             cmd="cd /home/cephuser/; git clone {}".format(
-                "https://github.com/vijaykumar-koppad/Crefi.git"
+                "https://github.com/yogesh-mane/Crefi.git"
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:63]: why have you used your fork here. I saw the change in your fork. you have removed `import xattr`. Try giving this change to the upstream repo `(https://github.com/vijaykumar-koppad/Crefi.git)`. Or else yuo will have to keep maintaining your fork always.

Comment (sunilkumarn417) [reply tests/cephfs/cephfs_utils.py:63]: Hi @yogesh-mane 

When we had last discussion with Hemanth Kumar and @rakeshgm, there was few talks on replacing smallfile over crefi. 
Is there any change of plan here?

Comment (rakeshgm) [reply tests/cephfs/cephfs_utils.py:63]: i guess this is not getting merged any time sooner: https://github.com/vijaykumar-koppad/Crefi/pull/5/files. 
@yogesh-mane fall back to your fork. lets merge it ASAP

Comment (yogesh-mane) [reply tests/cephfs/cephfs_utils.py:63]: Hi @rakeshgm , I have tested these changes.
4.2-rhel-7 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1605962194616
4.2-rhel-8 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1606131196375

**Pull request merged**